headers/types/common/graphics.h

```diff
@@ -71,4 +72,97 @@ struct animation_control {
 };
 ASSERT_SIZE(struct animation_control, 124);
 
+// Represent a single element to render using the 3D engine
+// This structure is used in two different way. The first one is for planning rendering, that will
+// then call function that will call a function that will then add themm to RENDER_3D for rendering
+// later in the frame.
+struct render_3d_element {
+    undefined2 render_function_id; // range from 0 to 3 (included)
+    undefined2 field1_0x2;         /* appears to be a render priority level. Impact sorting. */
+    undefined4 field2_0x4;
+    undefined4 field3_0x8;
+    uint16_t x_tileset_start;
+    uint16_t y_tileset_start;
+    uint16_t x_tileset_width_offset;
+    uint16_t y_tileset_height_offset;
+    undefined2 field8_0x14;
+    undefined2 x_start_1;
+    undefined2 y_start_1;
+    undefined2 x_end_1;
+    undefined2 y_start_2;
+    undefined2 x_start_2;
+    undefined2 y_end_1;
+    undefined2 x_end_2;
+    undefined2 y_end_2;
+    undefined2 field17_0x26[4];
+    undefined2 field18_0x2e;
+    undefined field19_0x30;
+    char field20_0x31;
+    bool field21_0x32;
+    undefined field22_0x33;
+};
+ASSERT_SIZE(struct render_3d_element, 52);
+
+// A global, unique structure that store element relating to the 3d engine, in particular the list
+// of element to render later in the frame.
+struct render_3d_global {
```

Comment:
I don't see a data symbol that uses this type. Did you forget to add `RENDER_3D` somewhere? You mention this struct a bunch in the descriptions.